spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/RestClientRibbonCommand.java

```diff
@@ -33,33 +34,32 @@
 /**
  * Hystrix wrapper around Eureka Ribbon command
  *
- * see original
- * https://github.com/Netflix/zuul/blob/master/zuul-netflix/src/main/java/com/
- * netflix/zuul/dependency/ribbon/hystrix/RibbonCommand.java
+ * see <a href="https://github.com/Netflix/zuul/blob/master/zuul-netflix/src/main/java/com/netflix/zuul/dependency/ribbon/hystrix/RibbonCommand.java">original</a>
  */
 @SuppressWarnings("deprecation")
 public class RestClientRibbonCommand extends AbstractRibbonCommand<RestClient, HttpRequest, HttpResponse> {
 
-	@SuppressWarnings("unused")
-	@Deprecated
-	public RestClientRibbonCommand(String commandKey, RestClient restClient, HttpRequest.Verb verb, String uri,
-								   Boolean retryable, MultiValueMap<String, String> headers,
-								   MultiValueMap<String, String> params, InputStream requestEntity) {
-		this(commandKey, restClient, new RibbonCommandContext(commandKey, verb.verb(), uri, retryable, headers, params, requestEntity));
+	public RestClientRibbonCommand(String commandKey, RestClient client,
+			RibbonCommandContext context, ZuulProperties zuulProperties) {
+		super(commandKey, client, context, zuulProperties);
 	}
 
-	public RestClientRibbonCommand(String commandKey, RestClient client, RibbonCommandContext context) {
```

Comment:
For me it was not used by spring cloud sleuth.
I tried this code against release 1.0.4 without any problem.